aquarium/src/cg/group4/view/AquariumScreen.java

```diff
@@ -185,36 +131,35 @@ protected void initTooltipLabels() {
     }
 
     /**
-     * Initializes the status label.
+     * Converts the id to s username.
+     *
+     * @param id The id.
+     * @return The username.
      */
-    protected void initStatusLabel() {
-        cStatusLabel = cStyle.generateDefaultLabel("");
-        cLabelTable.add(cStatusLabel).expandX();
+    public String convertToUsername(String id) {
+        if (cIdToUserName.containsKey(id)) {
+            return cIdToUserName.get(id);
+        }
+        return id;
     }
 
     /**
-     * To be executed only after initialization of the collectible renderer entities.
+     * Initializes the status label.
      */
-    public void initStage() {
-        for (CollectibleRenderer cr : cDisplayRendererSet) {
-            cStage.addActor(cr.getActor());
-        }
-        Gdx.input.setInputProcessor(cStage);
+    protected void initStatusLabel() {
+        cStatusLabel = cStyle.generateDefaultLabel("");
+        cLabelTable.add(cStatusLabel).expandX();
     }
 
-
     @Override
     public void show() {
     }
 
     @Override
     public void render(float delta) {
-
-        //System.out.println("Fps -> " + (1/delta));
         for (CollectibleRenderer c : cDisplayRendererSet) {
             c.render();
         }
-
         cStage.act();
 
         final float alpha = 1f;
```

Comment:
1 is not a magic number and should just be added to the method parameter directly